status_effects_service/status_effects_service.lua

```diff
@@ -15,39 +15,41 @@ status_effects.data = {
     party = {},
 }
 
-packets.incoming[0x063][9]:register(function(p)
-    for i = 1, 0x20 do
-        local buff_id = p.status_effects[i - 1]
-        if buff_id == 0 or buff_id == 0xFF then 
-            status_effects.data.player[i] = nil
-        else
-            status_effects.data.player[i] = {
-                id = buff_id,
-                timestamp = (p.durations[i - 1] / 60) + 501079520 + 1009810800 - os.time()
-            }
+packets.incoming:register_init({
+    [{0x063, 9}] = function(p)
+        for i = 1, 0x20 do
+            local buff_id = p.status_effects[i - 1]
+            if buff_id == 0 or buff_id == 0xFF then 
```

Comment:
White spaces after then :) looks like this one was missed in the earlier version.